### ISSUE TITLE
Excel needs more verbosity when selecting text within a cell in NVDA

### ISSUE THREAD

**Issue by Daredevilsprmcy** [nvaccess/nvda#7534]
1. Go to a spreadsheet file with filled in cells in Excel 2010.
2. Let's say that cell A23 has some text written in like Pajamas are not for sale.
3. Press F2 to go into cell A23.
4. Press and hold shift while pressing the right arrow to select character by character or ctrl+shift while pressing the right arrow to select word by word.
5. The selected items will not be announced, and they should.
6. You can still press ctrl+c or ctrl+x to copy or cut the selected portion of the text, but the lack in verbosity makes the entire process not very intuitive.
7. Thing to note is that if you're selecting several cells, and not specific text, NVDA announces everything clearly and fully.
8. Also, whenever you are in Google Sheets which is Google's online version of Excel, pressing F2, and selecting text by character or by word, NVDA reads everything as should.

**Comment by fisher729** [comment]
Hi @Daredevilsprmcy 
I cannot reproduce this issue. When I select text in a cell after pressing F2, all is announced for me. Which versions of Excel (office), NVDA and Windows are you using? Also, have you tried to accomplish this with add-ons disabled (choosing to restart with add-ons disabled should do the trick)

**Comment by Daredevilsprmcy** [comment]
I’m using Office 14 with Excel 2010, Windows 10 Creator’s Edition, NVDA v. 2017.3.

Which command do you use to start NVDA with all of the add-ons disabled again?

 

Thanks.

 

Dmitriy

**Comment by ehollig** [comment]
Thank you @Daredevilsprmcy for reporting this issue. However, this is a known issue as reported in #2946 with Excel 2010. Closing as duplicate.

**Comment by Daredevilsprmcy** [comment]
Hello,

 

I’ve restarted with all of the add-ons disabled, and for me it gives the same issue: when pressing f2 within a cell in Excel, then shift+right arrow, NVDA is not announcing what portions of text I have selected.

 

Dmitriy
